### PR TITLE
Remove response body from POST, DELETE, PATCH endpoints

### PR DESCRIPTION
**Pull request by mchompalova**

### CODE REVIEW

File: _ft/api.yml [hunk 1]
```diff
@@ -159,11 +159,6 @@ paths:
       responses:
         200:
           description: Returns a canonicalized array of annotations that have been successufully saved in PAC.
```
Comment (MiroslavGatsanoga) [inline _ft/api.yml]: The descriptions for all responses should be updated.

File: handler/handler.go [hunk 2]
```diff
@@ -166,11 +166,11 @@ func (h *Handler) saveAndReturnAnnotations(ctx context.Context, w http.ResponseW
 	}
 
 	w.Header().Set(annotations.DocumentHashHeader, newHash)
-	err = json.NewEncoder(w).Encode(newAnnotations)
-	if err != nil {
-		handleWriteErrors("Error encoding draft annotations response", err, writeLog, w, http.StatusInternalServerError)
-		return
-	}
+	// err = json.NewEncoder(w).Encode(newAnnotations)
```
Comment (MiroslavGatsanoga) [inline handler/handler.go]: Let's delete instead of commenting out the code.

File: handler/handler.go [hunk 3]
```diff
@@ -166,11 +166,11 @@ func (h *Handler) saveAndReturnAnnotations(ctx context.Context, w http.ResponseW
 	}
 
 	w.Header().Set(annotations.DocumentHashHeader, newHash)
```
Comment (MiroslavGatsanoga) [inline handler/handler.go]: Do we still want to set the header in the response?

Comment (mchompalova) [reply handler/handler.go]: I think so. We only delete the body.

File: _ft/api.yml (hunk 1 above)
Comment (mchompalova) [reply _ft/api.yml]: Yep, I forgot to do this.

**Pull request merged**